include/geos/operation/buffer/OffsetCurve.h

```diff
@@ -93,27 +86,27 @@ class GEOS_DLL OffsetCurve {
 private:
 
     // Members
-    const Geometry& inputGeom;
+    const geom::Geometry& inputGeom;
```

Comment:
what about doing the following (generalized to similar cases), to both reduce verbosity and amount of lines changed ?
```suggestion
    using Geometry = geom::Geometry;

    const Geometry& inputGeom;
```

Reply:
So use `using` as a [_type alias_](https://en.cppreference.com/w/cpp/language/type_alias) instead of a [_using declaration_](https://en.cppreference.com/w/cpp/language/namespace)?  Is this safer in some way?  

What is the scope of the alias?  Is it just the namespace within the `cpp` file including the header?  Or is it the entire namespace (seems unlikely, since how would other compilation units see it?)

Reply:
> What is the scope of the alias?

the class in which you put it

Not saying this is a good idea. Mostly for brainstorming